app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -47,23 +47,13 @@
     @Column(name = "operation_type")
     public String operation_type;
 
-/*
-    public static List<BgSendQueue> queue() {
-        return new Select()
-                .from(BgSendQueue.class)
-                .where("success = ?", false)
-                .orderBy("_ID asc")
-                .limit(20)
-                .execute();
-    }
-*/
     public static List<BgSendQueue> mongoQueue() {
         return new Select()
                 .from(BgSendQueue.class)
                 .where("mongo_success = ?", false)
                 .where("operation_type = ?", "create")
-                .orderBy("_ID desc")
-                .limit(30)
+                .orderBy("_ID asc")
```

Comment:
I believe that in the case of adults taking care of a child, data is
constantly uploaded.

But in any case, for people who don't want to have nightwatchpro, and did
not change that checkbox nothing will change.

On Tue, Dec 15, 2015 at 2:15 PM, AdrianLxM <notifications@github.com> wrote:

> In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/220#discussion_r47628597>
> :
>
> >      public static List<BgSendQueue> mongoQueue() {
> >          return new Select()
> >                  .from(BgSendQueue.class)
> >                  .where("mongo_success = ?", false)
> >                  .where("operation_type = ?", "create")
> > -                .orderBy("_ID desc")
> > -                .limit(30)
> > +                .orderBy("_ID asc")
>
> If one is only uploading the data once in a month, how comes the live
> results are suddenly that important?
>
> In such cases where adults need remote-monitoring (like severe sickness
> and the spouse cannot stay at home, hospital stays after accidents...)
> there usually is not much time and ease to do long syncs and setups.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/220/files#r47628597>
> .
>